sklearn/linear_model/passive_aggressive.py

```diff
@@ -110,6 +121,16 @@ def partial_fit(self, X, y, classes=None):
         -------
         self : returns an instance of self.
         """
+        if self.class_weight == 'balanced':
+            raise ValueError("class_weight 'balanced' is not supported for "
+                             "partial_fit. In order to use 'balanced' "
+                             "weights, use "
+                             "compute_class_weight('balanced', classes, y). "
```

Comment:
Yes, it would be more clear, I'll update this. I'd simply copied from SGD's `partial_fit` message. I can also update the other instances where this message appears around the code base in a separate PR.